src/main/java/uk/gov/ons/census/casesvc/service/CCSPropertyListedService.java

```diff
@@ -37,7 +41,14 @@ public void processCCSPropertyListed(ResponseManagementEvent ccsPropertyListedEv
         caseService.createCCSCase(
             ccsProperty.getCollectionCase().getId(), ccsProperty.getSampleUnit());
 
-    uacService.createUacQidLinkedToCCSCase(caze);
+    if (ccsProperty.getUac() == null) {
+      uacService.createUacQidLinkedToCCSCase(caze);
+      ccsToFieldService.convertAndSendCCSToField(caze);
+    } else {
+      UacQidLink uacQidLink = uacService.findByQid(ccsProperty.getUac().getQuestionnaireId());
```

Comment:
I can absolutely guarantee that the requirements will change and it'll be a case of "oh we sometimes don't bother linking and maybe we'll do it later" in which case this would error I guess... but I support what you've done here. It meets the current requirements as we understand them